README.md

```diff
@@ -88,85 +84,66 @@ Setting up the Encryptic development environment only takes a few minutes. To ru
 ### Part One: Install Encryptic
 
 If you're planning to contribute to the project's development, hit the fork button at the top of the page and move on to **step 2**.
-<br/>
+
 
 **1. Clone the Encryptic repository:**
 
+With SSH :
 ```bash
 $ git clone git@github.com:daed/encryptic.git
 ```
-<br/>
-
-**2. Navigate to project directory and checkout `dev` branch:**
 
+With HTTPS :
 ```bash
-$ cd encryptic
-
-# unless you're patching bugs in the latest release, switch to dev branch:
-$ git fetch && git checkout dev
+$ git clone https://github.com/encryptic-team/encryptic.git
 ```
-<br/>
 
-**3. Install Gulp:**
 
-(yarn instructions coming soon)
+**2. Navigate to project directory and install gulp and yarn globally:**
+
 ```bash
-$ npm install gulp
+$ cd encryptic
+
 $ npm install -g gulp
+$ npm install -g yarn
 ```
-<br/>
+
 
 **4. Install dependencies and build:**
 
 ```bash
 $ yarn
 ```
-- or -
+\- or -
 ```bash
 $ npm run setup
 ```
 
-<br/>
 
 **5. Start Encryptic:**
 
 ```bash
 $ gulp
 ```
 
-**6. Point a web broswer to http://localhost:9000**
+**6. Point a web browser to http://localhost:9000**
 
 ## Coding Style Guidelines
 
-We ask that you use either **plain JavaScript or the [Marionette.js](http://marionette.js/) framework** (for more details on the preferred coding style see [.editorconfig](https://github.com/Encryptic/Encryptic/blob/master/.editorconfig)). All experimental changes are pushed to the `dev` branch. Feature changes need to be done on either `dev` or a branch that uses `dev` as its parent.
-
-Localizations [are available here.](https://github.com/Encryptic/Encryptic/blob/dev/CONTRIBUTE.md)
+We ask that you use either **plain JavaScript or the [Marionette.js](http://marionette.js/) framework** (for more details on the preferred coding style see [.editorconfig](https://github.com/encryptic-team/Encryptic/blob/master/.editorconfig)).
```

Comment:
Again the mention about the dev branch (or its children) is important @Maks-s.

Reply:
Same as https://github.com/encryptic-team/encryptic/pull/30#discussion_r235199042 ;)